utils/cli/multisend.go

```diff
@@ -0,0 +1,225 @@
+package cli
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	authtx "github.com/cosmos/cosmos-sdk/x/auth/tx"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	commontypes "github.com/lavanet/lava/utils/common/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+const HotWalletFlagName = "hot-wallet"
+
+// NewMultiSendTxCmd returns a CLI command handler for creating a MsgMultiSend transaction.
+// For a better UX this command is limited to send funds from one account to two or more accounts.
+func NewMultiSendTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
prevent human error running the script twice by saving the last index in a local file and reading it before continuing unless specified directly, can also save there the tx hashes, will help us verify things if something fails